src/loopCreator.ts

```diff
@@ -10,24 +10,18 @@ import packageJsonTemplate from "./templates/package.json.squirrelly";
 import indexTemplate from "./templates/src/index.ts.squirrelly";
 import tsconfigTemplate from "./templates/tsconfig.json.squirrelly";
 
+const ldkVersion = "3.0.0-beta.3";
+
 interface IWebviewMessage<T> {
   command: string;
   payload: T;
 }
 
 interface LoopFormData {
-  ldkVersion: string;
   language: string;
   path: string;
   projectName: string;
   aptitudes: string[];
-  clipboardReason: string;
```

Comment:
I forgot to remove all these reason fields in the last release (where I updated to `3.0.0-beta.3` of the LDK.